backpack/core/derivatives/mseloss.py

```diff
@@ -101,10 +69,64 @@ def hessian_mat_prod(mat):
 
         return hessian_mat_prod
 
-    def check_input_dims(self, module):
+    def _checks(self, module):
+        self._check_input_dims(module)
+
+    def _make_distribution(self, subsampled_input):
+        """Make the sampling distribution for the NLL loss form of MSE.
+
+        The log probabiity of the Gaussian distribution is proportional to
+        ¹/₍₂𝜎²₎∑ᵢ₌₁ⁿ (xᵢ−𝜇)². Because MSE = ∑ᵢ₌₁ⁿ(Yᵢ−Ŷᵢ)², this is
+        equivalent for samples drawn from a Gaussian distribution with
+        mean of the subsampled input and variance √0.5.
+
+        Args:
+            subsampled_input: input after subsampling
+
+        Returns:
+            torch.distributions Normal distribution with mean of
+        the subsampled input and variance √0.5
+
+        """
+        return Normal(mean(subsampled_input), tensor(sqrt(0.5)))
```

Comment:
Is this correct? I think the distribution's mean should be `subsampled_input`, not `mean(subsampled_input)`

Reply:
Fixed in comment
@jabader97 [Fixed some commenting, removed mean from mse make distribution, moved…](https://github.com/f-dangel/backpack/pull/252/commits/97cb44f6a85d3bb5b7ec3de309d871fd18f67f8c)